src/main/java/com/openlattice/organizations/roles/HazelcastPrincipalService.java

```diff
@@ -375,6 +366,10 @@ private static Predicate findPrincipal( Principal p ) {
         return Predicates.equal( "principal", p );
     }
 
+    private static Predicate findPrincipals( Collection<Principal> principals ) {
+        return Predicates.in( "principal", principals.toArray( new Principal[] {} ) );
```

Comment:
Better to use the `PrincipalMapstore.PRINCIPAL_INDEX` constant

Reply:
:thumbsup: